skill_tagging/pipeline.py

```diff
@@ -115,9 +115,12 @@ class AddVideoBlockSkillVerificationComponent(VerificationPipelineBase, Pipeline
     """
     def run_filter(self, block, context):  # pylint: disable=arguments-differ
         """Pipeline Step implementing the Filter"""
-        skills = self.fetch_related_skills(block)
         usage_id = block.scope_ids.usage_id
-        if not skills or not self.should_run_filter() or usage_id.block_type != "video":
+        if usage_id.block_type != "video":
+            # avoid fetching skills for other xblocks
```

Comment:
@pkulkark Currently this class/pipeline is specifically for video blocks (the injected javascript/css/html files will only work for video blocks). If we manage to make the class generic to support all blocks then it makes sense, but it would be simpler to just add classes for the xblocks we want to support moving common parts to base class.

```python
class AddVideoBlockSkillVerificationComponent(VerificationPipelineBase, PipelineStep):
    """Adds verification component to video blocks.
```